3rd_party/3rd_party.sh

```diff
@@ -292,27 +201,5 @@ case `uname` in
         fi
         ;;
 
-    SunOS)
```

Comment:
There was a previous PR in the old repo to remove Solaris support but it missed this bit.  This is just removing a bit of dead code since I'm refactoring the file so much anyway.  It's not like this PR is sneaking in removal of support for a platform.

Reply:
Yeah, I get that :-) Just shedding a tear for the passing of an old friend...